src/main.rs

```diff
@@ -164,7 +183,7 @@ fn _get_required_by_recursive(
 }
 
 /// Given a package and an `avg::AVG`, returns true if the system is affected
```

Comment:
this comment is wrong now, should mention the fixed version instead.

Reply:
Updated!